utils/oscap.8

```diff
@@ -95,6 +95,11 @@ Use given CPE dictionary or language (auto-detected) for applicability checks. (
 Write XCCDF results into FILE.
 .RE
 .TP
+\fB\-\-thin-results\fR
+.RS
+Set OVAL/ARF results to Thin Results.
```

Comment:
We can provide better description of what are results with "Thin Results", similar to `--without-syschar`.
Like "Provide only the minimal amount of information in OVAL/ARF result files."